feature-toggles/feature-toggles-impl/src/main/java/com/duckduckgo/feature/toggles/impl/RealFeatureTogglesCallback.kt

```diff
@@ -53,3 +78,29 @@ class RealFeatureTogglesCallback @Inject constructor(
 internal enum class FeatureTogglesPixelName(override val pixelName: String) : Pixel.PixelName {
     EXPERIMENT_ENROLLMENT("experiment_enroll"),
 }
+
+@ContributesPluginPoint(
+    scope = AppScope::class,
+    boundType = TargetMatcherPlugin::class,
+)
+@Suppress("unused")
+private interface TargetMatcherPluginTrigger
+
+@ContributesMultibinding(AppScope::class)
+class LocaleToggleTargetMatcher @Inject constructor(
+    private val appBuildConfig: AppBuildConfig,
+) : TargetMatcherPlugin {
+    override fun matchesTargetProperty(target: Target): Boolean {
+        val country = appBuildConfig.deviceLocale.country.lowercase()
+        val language = appBuildConfig.deviceLocale.language.lowercase()
+
+        val isCountryMatching = target.localeCountry?.let {
+            country == it.lowercase()
+        } ?: true
+        val isLanguageMatching = target.localeLanguage?.let {
+            language == it.lowercase()
+        } ?: true
+
+        return isCountryMatching && isLanguageMatching
+    }
+}
```

Comment:
Looking at this logic, I realised we only support one language and one country as a target. If we wanted to test multiple languages, it wouldn’t work, right? Shouldn't it be an array instead of a String?
cc/ @marcosholgado

Reply:
it supports multiple languages, targets is an array already, i.e.

"targets": [
   {
      "variantKey": "mh"
   },
   {
      "variantKey": "mp"
   }
]

Reply:
@nalcalag Added a test in https://github.com/duckduckgo/Android/pull/5277/commits/7575d1eafa0763217e70abd473c6e74f75b78954 so that we can see how it works